From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Mon, 22 Nov 2021 15:59:49 +0800
Subject: content: gpu: Only depend dri for X11

Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>

diff --git a/content/gpu/BUILD.gn b/content/gpu/BUILD.gn
index 098082510efab71fb997d20c7534ee28619fbdf0..2b53ff60043049480893c7a4da6dab975e21e4d5 100644
--- a/content/gpu/BUILD.gn
+++ b/content/gpu/BUILD.gn
@@ -128,7 +128,7 @@ target(link_target_type, "gpu_sources") {
 
   # Use DRI on desktop Linux builds.
   if (current_cpu != "s390x" && current_cpu != "ppc64" && is_linux &&
-      !is_castos) {
+      !is_castos && ozone_platform_x11) {
     configs += [ "//build/config/linux/dri" ]
   }
 }
